src/contexts/wizard/WizardDataProvider.tsx

```diff
@@ -74,14 +65,6 @@ export default memo(function WizardDataProvider({ id, children }: Props) {
     setIsRefreshingNfts(false);
   }, [loadQuery, pushToast, syncTokens]);
 
-  useEffect(() => {
-    if (id === 'onboarding') {
-      void handleRefreshNfts();
-    }
-    // just want this running once for the welcome screen
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
```

Comment:
❗ previously, we began fetching the user's NFTs as soon as onboarding began, since we could assume they were logged in (without a username) by that point. however, we now need to wait to begin fetching until they select their username, which is further into the flow. this will result in less time for NFTs to be fetched before they reach the Collection Creation step.

the fetching now begins here: https://github.com/gallery-so/gallery/pull/883/files#diff-227e443aa0348b7881e6bd7e3256beb1c80f48335332b6764b002a5fae96326aR55